src/wireviz/Harness.py

```diff
@@ -20,16 +20,16 @@ def __init__(self):
         self.cables = {}
         self.additional_bom_items = []
 
-    def add_connector(self, name, *args, **kwargs):
+    def add_connector(self, name: str, *args, **kwargs) -> None:
         self.connectors[name] = Connector(name, *args, **kwargs)
 
-    def add_cable(self, name, *args, **kwargs):
+    def add_cable(self, name: str, *args, **kwargs) -> None:
         self.cables[name] = Cable(name, *args, **kwargs)
 
-    def add_bom_item(self, item):
+    def add_bom_item(self, item: str) -> None:
         self.additional_bom_items.append(item)
 
-    def connect(self, from_name, from_pin, via_name, via_pin, to_name, to_pin):
+    def connect(self, from_name: str, from_pin: int, via_name: str, via_pin: (int, str), to_name: str, to_pin: int) -> None:
```

Comment:
`from_pin` and `to_pin` can be `str` as well.

Reply:
Fixed in most recent push